pac/using-pipelines-as-code-repos.adoc

```diff
@@ -0,0 +1,63 @@
+:_content-type: ASSEMBLY
+[id="using-pipelines-as-code-repos"]
+include::_attributes/common-attributes.adoc[]
+= Using {pac} with a Git repository hosting service provider
+:context: using-pipelines-as-code-repos
+
+toc::[]
+[role="_abstract"]
+After installing {pac}, cluster administrators can configure a Git repository hosting service provider. Currently, the following services are supported:
+
+* GitHub App
+* GitHub Webhook
+* GitLab
+* Bitbucket Server
+* Bitbucket Cloud
+
+[NOTE]
+====
+GitHub App is the recommended service for using with {pac}.
```

Comment:
@mramendi Avoid the word recommended. Refer to https://redhat-documentation.github.io/supplementary-style-guide/#recommend